modules/labs-react/combobox/lib/AutocompleteList.tsx

```diff
@@ -7,8 +7,10 @@ import {ComboBoxMenuItemGroup, getOptionId, listBoxIdPart, getTextFromElement} f
 import uuid from 'uuid/v4';
 
 const Autocomplete = styled('ul')({
-  padding: 0,
-  margin: `${space.xxs} 0`,
+  margin: 0,
+  maxHeight: 184,
+  overflowY: 'scroll',
```

Comment:
I think this should be `overflowY: 'auto'` so we don't always see the scrollbar unless it's needed.

```suggestion
  overflowY: 'auto',
```